pkg/metrics/metrics.go

```diff
@@ -158,3 +208,31 @@ func (m *ServerMetrics) RegisterBackupDuration(backupSchedule string, seconds fl
 func toSeconds(d time.Duration) float64 {
 	return float64(d / time.Second)
 }
+
+// RegisterRestoreAttempt records an attempt to restore a backup.
+func (m *ServerMetrics) RegisterRestoreAttempt(backupSchedule string) {
+	if c, ok := m.metrics[restoreAttemptTotal].(*prometheus.CounterVec); ok {
+		c.WithLabelValues(backupSchedule).Inc()
+	}
+}
+
+// RegisterRestoreSuccess records a successful (maybe partial) completion of a restore.
+func (m *ServerMetrics) RegisterRestoreSuccess(backupSchedule string) {
```

Comment:
Would `RegisterRestoreCompleted` be a better name here?

Reply:
Maybe...it currently mirrors the backup success name, though.

IMO, if we change one we should change both. There's a TODO on L33 to update the strings to match the names, too.